fahclient/templates/statefulset.yaml

```diff
@@ -23,10 +23,12 @@ spec:
       priorityClassName: {{ .Values.priorityClassName }}
       {{- end }}
       terminationGracePeriodSeconds: 45
+      securityContext:
+        {{- toYaml .Values.securityContext | nindent 8 }}
       initContainers:
       - name: init
         image: alpine:latest
-        command: ['sh', '-c', "mkdir -p /fah && cp -f /tmp/fahclient/config.xml /fah/config.xml && chown -R {{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.runAsGroup }} /fah"]
+        command: ['sh', '-c', "cp -f /tmp/fahclient/config.xml /fah/config.xml"]
```

Comment:
I think the mkdir is still required on a new volume. But not sure. I'll test.

Reply:
I tested by destroying al PVCs and re-installing.  Shouldn't need this.  I've done the same "workaround" in other deployments without needing the `mkdir`

Reply:
Check for existing pvcs
````
kubectl get pvc -n fah
No resources found in fah namespace.
````
Check for exising pv
````
kubectl get pv | grep -i fahclient | wc -l
0
````
Run helm install (from this branch)
````
helm install fah . -n fah -f my-values.yaml --wait --debug
````
See all the things exist now and everything is happy!!
````
kubectl get pv | grep -i fahclient | wc -l
3

kubectl get pvc -n fah
NAME                  STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
fah-fah-fahclient-0   Bound    pvc-7002136e-472f-456c-a240-bc3f59bfaeb2   1Gi        RWO            nfs-client     7s
fah-fah-fahclient-1   Bound    pvc-433dce7c-a402-4e4e-b6f6-8bc1ee01e767   1Gi        RWO            nfs-client     7s
fah-fah-fahclient-2   Bound    pvc-852016ab-99f7-42ef-8e5f-5a01554972eb   1Gi        RWO            nfs-client     7s

kubectl get pods -n fah
NAME              READY   STATUS    RESTARTS   AGE
fah-fahclient-0   1/1     Running   0          2m17s
fah-fahclient-1   1/1     Running   0          2m17s
fah-fahclient-2   1/1     Running   0          2m17s
````